backend/fms_core/services/study.py

```diff
@@ -97,3 +99,15 @@ def get_study(project_obj: Project, study_letter: str):
         errors.append(f"Both a project and a study letter are required to retrieve a study.")
 
     return study, errors, warnings
+
+def delete_study_errors(study: int) -> Dict[str, List[str]]:
```

Comment:
rename it to tell what it actually do (is_study_used, is_study_referenced, can_remove_study, ...)